src/pilot/deploy-overcloud.py

```diff
@@ -623,6 +624,7 @@ def main():
               "environments/ceph-ansible/ceph-rgw.yaml" \
               " {}" \
               " --libvirt-type kvm" \
+              " --no-cleanup" \
```

Comment:
no issue with having this, but any reason ?

Reply:
Ah, I believe this was just for debugging, but that said, does it hurt to leave that behind for post-mortem, could be useful for debugging failed deployment?  Please advise.

Reply:
happy to leave that in if usefull for debugging

Reply:
Let's keep it then